resources/rafter/charts/controller-manager/charts/minio/templates/virtualservice.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: {{ template "minio.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "minio.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  gateways:
+    - {{ .Values.global.istio.gateway.namespace }}/{{ .Values.global.istio.gateway.name }}
+  hosts:
+    - storage.{{ .Values.global.ingress.domainName }}
+  http:
+    - corsPolicy:
+        allowHeaders:
+          - Authorization
+          - Content-Type
+          - '*'
+        allowMethods:
+          - GET
+          - POST
+          - PUT
+          - DELETE
+        allowOrigins:
+          - regex: .*
+      match:
```

Comment:
Previously, an Oathkeeper rule was configured to only match `GET` requests on any path. Now we allow all methods. A `method` parameter should be added. See: https://istio.io/v1.9/docs/reference/config/networking/virtual-service/#HTTPMatchRequest